common/annotations/en.xml

```diff
@@ -734,306 +734,306 @@ annotations.
 		<annotation cp="💕" type="tts">two hearts</annotation>
-		<annotation cp="💟">143 | emotion | heart | heart decoration | purple heart | white hearth</annotation>
+		<annotation cp="💟">143 | decoration | emotion | heart | hearth | purple | white</annotation>
 		<annotation cp="💟" type="tts">heart decoration</annotation>
-		<annotation cp="❣">exclamation | heart exclamation | heavy heart exclamation | mark | punctuation</annotation>
+		<annotation cp="❣">exclamation | heart | heavy | mark | punctuation</annotation>
 		<annotation cp="❣" type="tts">heart exclamation</annotation>
-		<annotation cp="💔">break | broken | broken heart | crushed | emotion | heartbroken | lonely | sad</annotation>
+		<annotation cp="💔">break | broken | crushed | emotion | heart | heartbroken | lonely | sad</annotation>
 		<annotation cp="💔" type="tts">broken heart</annotation>
-		<annotation cp="❤‍🔥">burn | heart | heart on fire | love | lust | sacred heart</annotation> <!-- 2764 200D 1F525 -->
+		<annotation cp="❤‍🔥">burn | fire | heart | love | lust | sacred</annotation> <!-- 2764 200D 1F525 -->
 		<annotation cp="❤‍🔥" type="tts">heart on fire</annotation>
-		<annotation cp="❤‍🩹">healthier | improving | mending | mending heart | recovering | recuperating | well</annotation> <!-- 2764 200D 1FA79 -->
+		<annotation cp="❤‍🩹">healthier | heart | improving | mending | recovering | recuperating | well</annotation> <!-- 2764 200D 1FA79 -->
 		<annotation cp="❤‍🩹" type="tts">mending heart</annotation>
-		<annotation cp="❤">emotion | heart | love | red heart</annotation>
+		<annotation cp="❤">emotion | heart | love | red</annotation>
 		<annotation cp="❤" type="tts">red heart</annotation>
 		<annotation cp="🩷">143 | adorable | cute | emotion | heart | ily | like | love | pink | special | sweet</annotation> <!-- 1FA77 -->
 		<annotation cp="🩷" type="tts">pink heart</annotation>
-		<annotation cp="🧡">orange | orange heart</annotation>
+		<annotation cp="🧡">heart | orange</annotation>
 		<annotation cp="🧡" type="tts">orange heart</annotation>
 		<annotation cp="💛">143 | cardiac | emotion | heart | ily | love | yellow</annotation>
 		<annotation cp="💛" type="tts">yellow heart</annotation>
 		<annotation cp="💚">143 | emotion | green | heart | ily | love | romantic</annotation>
 		<annotation cp="💚" type="tts">green heart</annotation>
 		<annotation cp="💙">143 | blue | emotion | heart | ily | love | romance</annotation>
 		<annotation cp="💙" type="tts">blue heart</annotation>
-		<annotation cp="🩵">143 | cute | cyan | emotion | heart | ily | light blue | light blue heart | like | love | sky blue | special | teal</annotation> <!-- 1FA75 -->
+		<annotation cp="🩵">143 | blue | cute | cyan | emotion | heart | ily | light | like | love | sky | special | teal</annotation> <!-- 1FA75 -->
 		<annotation cp="🩵" type="tts">light blue heart</annotation>
 		<annotation cp="💜">143 | bestest | emotion | heart | ily | love | purple</annotation>
```

Comment:
shouldn't we always include 143 for all heart emojis or not (similar to how face is included for all the face emojis?)

Reply:
Added 143 to orange, white, and brown hearts in spreadsheet.

Reply:
You could compare https://docs.google.com/spreadsheets/d/1Pmhnui7OpF6lES3GUECkOgukcpRZ170UELC_v1GlTFM/edit#gid=1945807313&range=11:11
with 
https://docs.google.com/spreadsheets/d/1Pmhnui7OpF6lES3GUECkOgukcpRZ170UELC_v1GlTFM/edit#gid=1945807313&range=40:40
and we could fix the diff

Reply:
Just fixed in the source. I didn't add 143 to 'black heart' since that's associated with evil. WDYT?

Reply:
I'm good with that!